src/react-components/2d-hud.js

```diff
@@ -7,15 +7,21 @@ import styles from "../assets/stylesheets/2d-hud.scss";
 const TwoDHUD = ({ muted, frozen, spacebubble, onToggleMute, onToggleFreeze, onToggleSpaceBubble }) => (
   <div className={styles.container}>
     <div className={cx("ui-interactive", styles.panel, styles.left)}>
-      <div className={cx(styles.iconButton, styles.mute, { [styles.active]: muted })} onClick={onToggleMute} />
+      <div
+        className={cx(styles.iconButton, styles.mute, { [styles.active]: muted })}
+        title={muted ? "Unmute Mic" : "Mute Mic"}
```

Comment:
I can see the word `Mute` and mistakingly thinking this will mute the audio, not my microphone (though that's obviously not what is meant here, heh)